packages/@romejs/cli-flags/Parser.ts

```diff
@@ -20,6 +20,7 @@ import {Dict} from '@romejs/typescript-helpers';
 import {markup} from '@romejs/string-markup';
 import {descriptions} from '@romejs/diagnostics';
 import {JSONObject} from '@romejs/codec-json';
+import {commandCategories} from '@romejs/core/common/commands';
```

Comment:
I exported `commandCategories` from `@romejs/core` because it belongs to the public API.

```suggestion
import {commandCategories} from '@romejs/core';
```

However, doing so crashes Rome:
```
/private/var/folders/gp/z7cw34td69v0qlsytclcm9600000gn/T/rome-dev/index.js:76608
    ___R$project$rome$$romejs$cli$diagnostics$DiagnosticsPrinter_ts$DEFAULT_PRINTER_FLAGS,
    ^

ReferenceError: Cannot access '___R$project$rome$$romejs$cli$diagnostics$DiagnosticsPrinter_ts$DEFAULT_PRINTER_FLAGS' before initialization
    at /private/var/folders/gp/z7cw34td69v0qlsytclcm9600000gn/T/rome-dev/index.js:76608:5
    at Object.<anonymous> (/private/var/folders/gp/z7cw34td69v0qlsytclcm9600000gn/T/rome-dev/index.js:106453:3)
    at Module._compile (internal/modules/cjs/loader.js:1147:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1167:10)
    at Module.load (internal/modules/cjs/loader.js:996:32)
    at Function.Module._load (internal/modules/cjs/loader.js:896:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47
```